OneCommunityGlobal/HGNRest
# Jinchao add weekly summary indicator on leaderboard for admin users

# Description
![image](https://github.com/OneCommunityGlobal/HGNRest/assets/94319381/d6622996-7e86-473a-918b-b7dd2f6d1be2)

## Related PRS (if any):
This backend PR is related to frontend [#880](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/880)

## Main changes explained:
Update function `getLeaderboard` and `getUserLaborData` to provide the weekly summary submission state of latest week for each user data in leaderboard.

## How to test:
Please see refer FE PR [#880](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/880)
